src/components/Modal/Modal.js

```diff
@@ -417,7 +417,7 @@ export default class Modal extends Component {
                   kind={danger ? 'danger--primary' : 'primary'}
                   disabled={primaryButtonDisabled}
                   onClick={onRequestSubmit}
-                  inputref={this.button}>
+                  inputRef={this.button}>
```

Comment:
I think on the button component the variable also needs to be changed.

Reply:
done